src/sentry/runner/initializer.py

```diff
@@ -188,14 +188,15 @@ def configure_structlog():
 
     structlog.configure(**kwargs)
 
-    lvl = os.environ.get('SENTRY_LOG_LEVEL', None)
+    lvl = os.environ.get('SENTRY_LOG_LEVEL', 'INFO')
     if lvl in logging._levelNames:
```

Comment:
Since this has passed the decorator onto the initializer, all other commands now run through this so either we decorate every single command or we just keep this sanity check here.